corehq/apps/export/models/export_settings.py

```diff
@@ -12,6 +14,19 @@ class ExportFileType(object):
         (EXCEL_PRE_2007, ugettext_lazy("Excel (older versions)")),
     )
 
+    @classmethod
+    def map_to_couch_type(cls, filetype):
+        from couchexport.models import Format
+
+        if filetype == ExportFileType.EXCEL_2007_PLUS:
+            return Format.XLS_2007
+        elif filetype == ExportFileType.EXCEL_PRE_2007:
+            return Format.XLS
+        elif filetype == ExportFileType.CSV:
+            return Format.CSV
+        else:
+            raise ValueError(f"{filetype} is not supported for export")
+
```

Comment:
Not thrilled about this and I should have had better foresight when creating `DefaultExportSettings`. I see a couple of paths foward:
- the one implemented above - map these types to format strings used in couch
- update/migrate `DefaultExportSettings` to directly store these values instead of the `ExportFileType` choices I created

Reply:
I would call this method `get_file_type`

Reply:
I'd lean towards doing the migration, since it sounds pretty straightforward and would eliminate this code.

Reply:
I also prefer doing the migration now and removing the mapping. Just to be clear about what this entails, this file in the `export` app would need to reference code in ex-`submodules/couchexport` [here](https://github.com/dimagi/commcare-hq/blob/579b52f1037d30c8e9184214f7278bb1d6a8b9d2/corehq/ex-submodules/couchexport/models.py#L8-L17). I'm assuming this is fine but just want to make sure. 

@biyeun are you opposed to the migration at all?

Reply:
I think that import is fine, especially given that [this one](./corehq/apps/export/models/new.py) exists.

Reply:
migration sounds like a good plan

Reply:
Created [this PR](https://github.com/dimagi/commcare-hq/pull/28982) to do the migration.